go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550
 	google.golang.org/api v0.26.0
 	google.golang.org/genproto v0.0.0-20200331122359-1ee6d9798940
+	google.golang.org/grpc v1.28.0
```

Comment:
Where does this come from?

Reply:
`go mod tidy` puts it there.

Reply:
Hmm, don't know why, as it's only an indirect dependency via cloud.google.com/go and it's already [in their `go.mod`](https://code.googlesource.com/gocloud/+/refs/tags/v0.56.0/go.mod#27).

I'm getting the same result with `go mod tidy` though.

Let's include it in for now, as I don't see any problems with it, as it's the same pinned version as is defined in the dep.